Framework/PythonInterface/mantid/api/src/Exports/IEventList.cpp

```diff
@@ -67,6 +73,8 @@ void export_IEventList() {
       .def("maskTof", &IEventList::maskTof, args("self", "tofMin", "tofMax"),
            "Mask out events that have a tof between tofMin and tofMax "
            "(inclusively)")
+      .def("maskCondition", &maskCondition, args("self", "mask"),
```

Comment:
Can we add some tests for the Python export behaviour? Maybe in [`EventListTest`](https://github.com/mantidproject/mantid/blob/master/Framework/PythonInterface/test/python/mantid/dataobjects/EventListTest.py)?

Reply:
Added a test to the python event list test file.